File: day66/classwork/cw.py
while True:
    try:
        num1 = float(input("შეიყვანეთ პირველი რიცხვი: "))
        num2 = float(input("შეიყვანეთ მეორე რიცხვი: "))
        print(f"ნამრავლი: {num1 * num2}")
    except ValueError:
        print("გთხოვთ, შეიყვანეთ მხოლოდ რიცხვები.")
        continue

    choice = input("თუ გსურთ პროგრამიდან გასვლა, შეიყვანეთ 'exit'. გაგრძელებისთვის დააჭირეთ Enter: ")
    if choice.lower() == 'exit':
        print("პროგრამა დასრულდა.")
        break

File: day112/classwork/cw.js
function calculate(num1, operation, num2) {
  switch (operation) {
    case '+':
      return num1 + num2;
    case '-':
      return num1 - num2;
    case '*':
      return num1 * num2;
    case '/':
      if (num2 === 0) {
        return null;
      }
      return num1 / num2;
    default:
      return null;
  }
}